Formula/m/mbpoll.rb

```diff
@@ -0,0 +1,32 @@
+class Mbpoll < Formula
+  desc "Command-line utility to communicate with ModBus slave (RTU or TCP)"
+  homepage "https://epsilonrt.fr"
+  url "https://github.com/epsilonrt/mbpoll/archive/refs/tags/v1.5.2.tar.gz"
+  sha256 "7d960cd4459b5f7c2412abc51aba93a20b6114fd75d1de412b1e540cfb63bfec"
+  license "GPL-3.0-only"
+
+  head "https://github.com/epsilonrt/mbpoll.git", branch: "master"
+
+  depends_on "cmake" => :build
+  depends_on "make" => :build
+  depends_on "pkg-config" => :build
+  depends_on "libmodbus" => ["3.1.4"]
+
+  def install
+    mkdir "build" do
+      system "cmake", "-G", "Unix Makefiles", "..", *std_cmake_args
+      system "make"
+      system "make", "install"
+    end
+    bin.install "build/mbpoll"
+  end
+
+  test do
+    # NOTE: using "1.0-0" and not "1.5.2" due to open PR: https://github.com/epsilonrt/mbpoll/pull/58
+    assert_match "1.0-0", shell_output("#{name} -V")
+
+    invalid_request = "#{name} -1 -o 0.01 -q -m tcp invalid.host 2>&1"
+    assert_match "", shell_output(invalid_request, 1)
```

Comment:
```suggestion
    assert_empty, shell_output(invalid_request, 1)
```

Reply:
note, there was minor error, an extra comma.  Changed this:
`assert_empty, shell_output(invalid_request, 1)` to:
`assert_empty shell_output(invalid_request, 1)`

Reply:
sorry about the extra comma, but can you use `assert_empty` rather than `assert_match "",`? (the latter is wrong)